result/convergence.yaml

```diff
@@ -0,0 +1,102 @@
+components:
+  schemas:
+    Convergence.Request:
+      description: >-
+        The container for a CPDP convergence request.
+      type: object
+      properties:
+        flow_names:
+          description: |-
+            Flow metrics will be retrieved for these flow names.
+            If no flow names are specified then all flows will be returned.
+          type: array
+          items:
+            type: string
+          x-constraint:
+          - "/components/schemas/Flow/properties/name"
+          x-field-uid: 1
+    Convergence.Metric:
+      description: |-
+        The container for convergence measurements
+      type: object
+      properties:
+        name:
+          description: >-
+            The name of a flow. 
+          type: string
+          x-field-uid: 1
+        data_plane_convergence_us:
+          description: |-
+            The convergence time measured from the data plane perspective only. 
+            It measures the time from BTT, when the rate on Test Port 2 crosses 
+            below the Rx Threshold until an acceptable amount of traffic was 
+            received at time ATT, when the rate crosses above the configured 
+            rx_rate_threshold.
+          type: number
+          format: double
+          x-field-uid: 2
+        control_plane_data_plane_convergence_us:
+          description: |-
+            The total convergence time, between the event that caused the
+            switchover (at tEvent) until an acceptable amount of traffic was 
```

Comment:
what is tEvent ?

Reply:
tEvent: timestamp of the event triggering the convergence.
Since we are mentioning "the event that caused the switchover", this can be omitted. 
@apratimmukherjee